module/EntraBeta/Microsoft.Entra.Beta/Applications/Set-EntraBetaApplicationProxyApplication.ps1

```diff
@@ -115,10 +118,10 @@ function Set-EntraBetaApplicationProxyApplication {
         if ($null -ne $PSBoundParameters["ConnectorGroupId"]) {
             $ConnectorGroupId = $PSBoundParameters["ConnectorGroupId"]
             $ConnectorGroupBody = @{
-                "@odata.id" = "https://graph.microsoft.com/beta/onPremisesPublishingProfiles/applicationproxy/connectorGroups/$ConnectorGroupId"
+                "@odata.id" = "$rootUri/beta/onPremisesPublishingProfiles/applicationproxy/connectorGroups/$ConnectorGroupId"
```

Comment:
Just a double-check: is the `$rootUri` still required in the ConnectorGroupBody for the following Graph request? I think it may be, even though the Uri doesn't require the base URI. Again, just double-checking for my own understanding.